docs/source/faq.rst

```diff
@@ -0,0 +1,113 @@
+==========================
+Frequently Asked Questions
+==========================
+
+This page contains frequently asked questions about MOTECH with short answers to them.
+
+.. contents::
+    :depth: 2
+
+Can MOTECH run on Windows?
+--------------------------
+MOTECH is avaliable on Windows and it is possible to run the service on it, but it is not actively supported.
+
+Is MDS using any caching? How does that affect a cluster deployment?
+--------------------------------------------------------------------
+MDS is using datanucleus, which by default uses level1 and level2 cache. Sometimes it can cause data inconsistency
```

Comment:
the DataNucleus persistence framework (maybe add a link to datanucleus, probably also good to add link to the caching doc on the datanucleus site)